components/atoms/button/button.tsx

```diff
@@ -167,15 +167,7 @@ export const Button = ({
     <button className={buttonStyles} onClick={onClick} type={type}>
       {(leftIconSrc || isLoading) && (
         <div className={iconWrapperStyles}>
-          {isLoading && <LoadingIcon />}
-          {leftIconSrc && (
-            <Image
-              src={leftIconSrc}
-              alt="left icon"
-              width={iconSize}
-              height={iconSize}
-            />
-          )}
+          {isLoading ? (
```

Comment:
아 제가 suggestion을 그냥 한줄을 클릭하고 했는데 그게 그대로 들어갔군요!!
제가 suggestion 으로 써드린거는 <div>태그 내부 전체에 관한 코드였습니다 ㅎ

leftIconSrc || isLoading 이란 조건으로 이미 iconWrapper를 띄울지 말지를 분기처리하고 있기 때문에, 내부는 단순히 isLoading 이면 -> 로딩상태 / leftIconSrc 이면 -> 아이콘 
이렇게만 나누어주면 돼서 삼항연산자보다는 &&가 개인적으로는 가독성에 더 좋아보였어요~~
제가 제시드리는 총 코드는 다음과 같겠네요!

```suggestion
{(leftIconSrc || isLoading) && (
        <div className={iconWrapperStyles}>
          {isLoading && <LoadingIcon />}
          {leftIconSrc && (
            <Image
              src={leftIconSrc}
              alt="left icon"
              width={iconSize}
              height={iconSize}
            />
          )}
            <LoadingIcon />
          ) : (
            leftIconSrc && (
              <Image
                src={leftIconSrc}
                alt="left icon"
                width={iconSize}
                height={iconSize}
              />
            )
          )}
        </div>
      )}
```

이전 코드를 보니까 제가 드린 suggestion이 제가 클릭한 줄에 그대로 들어가고 그 뒤에 부분도 그대로 들어갔는데, iconWrapper div 내부에 들어갈 전체 코드를 제시 드린거였어요~~

Reply:
앗 그런거였군요! 리뷰한 부분대로 커밋했는데 div 태그들이 맞지 않아서 이상하다 싶었어요 🥲 감사합니다~